api/db/schema.rb

```diff
@@ -27,9 +32,22 @@
     t.json "course_warnings", default: [], array: true
     t.json "warnings", default: [], array: true
     t.integer "course_counter"
+    t.boolean "is_currently_being_edited", default: false, null: false
     t.index ["user_id"], name: "index_plans_on_user_id"
   end
 
+  create_table "template_plans", force: :cascade do |t|
+    t.string "name", null: false
+    t.integer "catalog_year", null: false
+    t.json "schedule"
+    t.string "major", null: false
+    t.string "plan_string", null: false
```

Comment:
Let's switch plan_string to coop_cycle, since now we have "plans" which are schedules that a user has